ibis/backends/clickhouse/compiler.py

```diff
@@ -84,14 +84,35 @@ def get_result(self):
             self.join_types, self.join_tables[1:], self.join_predicates
         ):
             buf.write('\n')
-            buf.write(util.indent('{0} {1}'.format(jtype, table), self.indent))
-
-            if len(preds):
+            buf.write(util.indent(f'{jtype} {table}', self.indent))
+
+            fmt_preds = []
+            npreds = len(preds)
+            using_clause_flag = True
+            for pred in preds:
+                op = pred.op()
+                if (
+                    not isinstance(op, ops.Equals)
+                    or op.args[0].get_name() != op.args[1].get_name()
+                ):
+                    using_clause_flag = False
```

Comment:
Can we avoid using the `USING` keyword here? It's adding a ton of complexity to this code and if it isn't necessary then we shouldn't use it.

Reply:
When stop using the `USING` keyword, there is no need to override this method, so i remove it and some othere duplicated codes.

Reply:
Awesome. Loving the code deletion.